packages/console/src/pages/RoleDetails/index.tsx

```diff
@@ -83,6 +93,25 @@ function RoleDetails() {
       className={classNames(isPageHasTable && styles.withTable)}
       onRetry={mutate}
     >
+      {/* Todo @xiaoyijun remove dev feature flag */}
+      {isDevFeaturesEnabled && !isM2mRoleNotificationAcknowledged && (
+        <InlineNotification
+          action="general.got_it"
+          onClick={() => {
+            void updateConfigs({
+              m2mRoleNotificationAcknowledged: true,
+            });
```

Comment:
Need to double check with designer if this needs to be stored as tenant config or user preference.

Reply:
per offline discussion, store in the user preference, updated.